src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  box-sizing: border-box;
+  max-width: 200px;
+  width: 100%;
+  padding: 32px 16px 16px;
+  border: 1px solid #F3F3F3;
+
+  &__img {
+    max-width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+
+    img {
+      display: block;
+      object-fit: cover;
+      object-position: center;
+      width: 100%;
+      height: 100%;
+    }
+  }
+
+  &__title {
+    font-size: inherit;
```

Comment:
redundant
```suggestion
```

Reply:
The `.card__title` is an `h2` element, therefore it does not inherit `font-size` from `body`. Should i bring it to `*` selector or just change the value to `12px ` in the same line?

Reply:
no needs to move to `*`. 
Here you can see that `h2` got a correct font-family
<img width="335" alt="Screenshot 2024-02-19 at 21 23 17" src="https://github.com/mate-academy/layout_product-cards/assets/117587463/6cbfa39b-e50a-4df7-a658-5378b57a163d">